Containerfile.common

```diff
@@ -61,6 +61,17 @@ RUN --mount=type=cache,dst=/var/cache/dnf \
     done && \
     find /var/cache/rpms
 
+# Remove kernel version from kmod package names
+# FIXME: This will only work for 6.* kernels unfortunately
+# FIXME: The sed is a gross hack, maybe PR upstream?
+RUN set -e; \
+    sed -i -e 's/args = \["rpmbuild", "-bb"\]/args = \["rpmbuild", "-bb", "--buildroot", "#{build_path}\/BUILD"\]/g' /usr/local/share/gems/gems/fpm-*/lib/fpm/package/rpm.rb; \
+    for rpm in $(find /var/cache/rpms/kmods -type f -name \*.rpm); do \
+        basename=$(basename ${rpm}); \
+        name=${basename%%-6*}; \
+        fpm --verbose -s rpm -t rpm -p ${rpm} -f --name ${name} ${rpm}; \
```

Comment:
I don't believe this is working as intended.

If you check the "Build Test Image" step (or build locally and inspect files in the resulting container image) you can see the `kmod-*.rpm` files still have the kernel version in them.

Here is evidence in an online log: https://github.com/ublue-os/akmods/actions/runs/11427517843/job/31791745978#step:13:895

```
+ dnf install -y /tmp/akmods-rpms/kmods/kmod-kvmfr-6.11.3-200.fc40.x86_64-0.0.git.24.a7102df3-1.fc40.x86_64.rpm /tmp/akmods-rpms/kmods/kmod-openrazer-6.11.3-200.fc40.x86_64-100.0.0.git.530.886f986d-1.fc40.x86_64.rpm /tmp/akmods-rpms/kmods/kmod-v4l2loopback-6.11.3-200.fc40.x86_64-0.13.2-1.fc40.x86_64.rpm /tmp/akmods-rpms/kmods/kmod-wl-6.11.3-200.fc40.x86_64-6.30.223.271-53.fc40.x86_64.rpm /tmp/akmods-rpms/kmods/kmod-xone-6.11.3-200.fc40.x86_64-0.0.git.152.60d92082-1.fc40.x86_64.rpm '/tmp/akmods-rpms/kmods/kmod-xpadneo-6.11.3-200.fc40.x86_64-0.9.6^20240923git70ef8ee-3.fc40.x86_64.rpm'
```

From a local where I copy FROM my local akmods common build to a simple container `COPY --from=localhost/akmods:pr-257-main-40 / /tmp/akmods/40` as a test
```
$ podman run --rm -ti foo
[root@1c9e8182bde5 /]# find /tmp/akmods/
/tmp/akmods/
/tmp/akmods/40
/tmp/akmods/40/rpms
/tmp/akmods/40/rpms/akmods
/tmp/akmods/40/rpms/akmods/akmod-kvmfr-0.0.git.24.a7102df3-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/akmods/akmod-openrazer-100.0.0.git.530.886f986d-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/akmods/akmod-v4l2loopback-0.13.2-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/akmods/akmod-wl-6.30.223.271-53.fc40.x86_64.rpm
/tmp/akmods/40/rpms/akmods/akmod-xone-0.0.git.152.60d92082-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/akmods/akmod-xpadneo-0.9.6^20240923git70ef8ee-3.fc40.x86_64.rpm
/tmp/akmods/40/rpms/kmods
/tmp/akmods/40/rpms/kmods/kmod-kvmfr-6.11.3-200.fc40.x86_64-0.0.git.24.a7102df3-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/kmods/kmod-openrazer-6.11.3-200.fc40.x86_64-100.0.0.git.530.886f986d-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/kmods/kmod-v4l2loopback-6.11.3-200.fc40.x86_64-0.13.2-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/kmods/kmod-wl-6.11.3-200.fc40.x86_64-6.30.223.271-53.fc40.x86_64.rpm
/tmp/akmods/40/rpms/kmods/kmod-xone-6.11.3-200.fc40.x86_64-0.0.git.152.60d92082-1.fc40.x86_64.rpm
/tmp/akmods/40/rpms/kmods/kmod-xpadneo-6.11.3-200.fc40.x86_64-0.9.6^20240923git70ef8ee-3.fc40.x86_64.rpm
/tmp/akmods/40/rpms/ublue-os
/tmp/akmods/40/rpms/ublue-os/ublue-os-akmods-addons-0.6-1.fc40.noarch.rpm
/tmp/akmods/40/rpms/ucore
```

Reply:
Yes that was the intended behavior so as to not induce a breaking change. The intended result is to remove the version name from the package name as it is shown in the dnf database.

Reply:
Ah! Thank you!

Yes, I see that now... comparing a kmod I have in my local host to the recent build:
```
$ rpm -qi kmod-xone-6.11.3-200.fc40.x86_64-0.0.git.152.60d92082-1.fc40.x86_64|grep -E "^Name|^Source"
Name        : kmod-xone-6.11.3-200.fc40.x86_64
Source RPM  : kmod-xone-6.11.3-200.fc40.x86_64-0.0.git.152.60d92082-1.fc40.src.rpm
```
```
[root@1c9e8182bde5 /]# rpm -qi /tmp/akmods/40/rpms/kmods/kmod-xone-6.11.3-200.fc40.x86_64-0.0.git.152.60d92082-1.fc40.x86_64.rpm|grep -E "^Name|^Source"
Name        : kmod-xone
Source RPM  : kmod-xone-0.0.git.152.60d92082-1.fc40.src.rpm
```

That's better than I'd realized. And... previously was more broken than I'd realized. 

Thank you for the clarification.